.github/workflows/ci.yaml

```diff
@@ -342,6 +342,7 @@ jobs:
     needs:
       - build_docker_images
     strategy:
+      fail-fast: false
```

Comment:
Why is this needed?

Reply:
@TomerShor just noticed that we don't have it here, so if one of matrix's run fail, all others will be cancelled which I believe is not expected behaviour

Reply:
Alright 👍